_pages/policies/employee-resources-policies/hiring-authorities.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Hiring Authorities
+tags:
+- Hiring
+- hiring
+---
+This page explains the different hiring path available to TTS
```

Comment:
I'm not sure it matters, but I think it should be hiring path_s_, right?

Reply:
yes